core/packages/contracts/src/test/BeefyClient.t.sol

```diff
@@ -51,67 +53,44 @@ contract BeefyClientTest is Test {
 
         // Allocate for input variables
         string[] memory inputs = new string[](10);
-        inputs[0] = "test/ffiWrapper.js";
-        // Always add type of command as first arguments
-        inputs[1] = "GenerateInitialSet";
+        inputs[0] = "node_modules/.bin/ts-node";
```

Comment:
@yrong I simplified the compilation of the ffiWrapper.ts script, by using `ts-node`, which automatically compiles and runs the script. This means we no longer need to compile the script manually via the package.json script.